modules/post/windows/gather/credentials/smartermail.rb

```diff
@@ -106,12 +138,47 @@ def get_smartermail_creds(path)
     end
 
     username = data.match(/<sysAdminUserName>(.+)<\/sysAdminUserName>/)
-    password = data.match(/<sysAdminPassword>(.+)<\/sysAdminPassword>/)
+    password = data.scan(/<(sysAdminPassword|sysAdminPasswordHash)>(.+)<\/(sysAdminPassword|sysAdminPasswordHash)>/).flatten[1]
+
     result['username'] = username[1] unless username.nil?
-    result['password'] = decrypt_des(Rex::Text.decode_base64(password[1])) unless password.nil?
+
+    if password
+      begin
+        result['password'] = decrypt_des(Rex::Text.decode_base64(password))
+      rescue OpenSSL::Cipher::CipherError
+        result['password'] = password
```

Comment:
The credential shouldn't be saved as ```private_type: :password``` when decryption fails. Probably should be saved as ```:nonreplayable_hash``` and provide the ```:jtr_format``` option to allow John the Ripper cracking?